docs/release-notes/8.7.asciidoc

```diff
@@ -10,6 +10,15 @@
 ==== Known issues
 * After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
 
+
+* The {agent} upgrade from 8.6.2 to 8.7.0 might fail for {agents} running the Osquery Manager integration ({agents} can get stuck in the `Updating` state) (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after you finish upgrading.
```

Comment:
```suggestion
* The {agent} upgrade from 8.6.2 to 8.7.0 or later might fail for {agents} running the Osquery Manager integration ({agents} can get stuck in the `Updating` state) (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after you finish upgrading.
```

Reply:
@nastasha-solomon I'm not sure we need "or later" here. All we really know about right now is 8.7.0, and hopefully this gets fixed in 8.7.1? But if it doesn't, then we just reprint this same Known Issue in the 8.7.1 release notes.